### PR TITLE
Fix double-click issue

### PR DESCRIPTION
**Pull request by GioSensation**
**Reviewer:** @alistairjcbrown 
**Asana:** https://app.asana.com/0/0/1204520802024813/f

## Description
Fixes a potential issue on desktop apps. More info on Asana.

## Steps to test
See Asana task.

### CODE REVIEW

File: integration-test/helpers/pages.js [hunk 1]
```diff
@@ -486,19 +486,25 @@ export function loginPage (page, opts = {}) {
             }
         },
         /**
-         * This is here to capture EXISTING functionality of `macOS` in production and prevent
-         * accidental changes to how `showAutofillParent` messages are sent
          * @returns {Promise<void>}
          */
-        async assertClickAndFocusMessages () {
+        async assertClickMessage () {
```
Comment (GioSensation) [inline integration-test/helpers/pages.js:491]: This stuff now only fires one message, either the click or focus, so I've updated the test cases.

File: src/UI/controllers/HTMLTooltipUIController.js [hunk 2]
```diff
@@ -187,6 +187,9 @@ export class HTMLTooltipUIController extends UIController {
             e.preventDefault()
             e.stopImmediatePropagation()
 
+            const isMainMouseButton = e.button === 0
+            if (!isMainMouseButton) return
+
```
Comment (GioSensation) [inline src/UI/controllers/HTMLTooltipUIController.js:192]: Unrelated, just makes it easier to open the devtools in debug builds.

File: src/UI/controllers/OverlayUIController.js [hunk 3]
```diff
@@ -75,15 +75,19 @@ export class OverlayUIController extends UIController {
         })
         this._mutObs.observe(document.body, {childList: true, subtree: true})
 
+        const position = getPosition()
```
Comment (GioSensation) [inline src/UI/controllers/OverlayUIController.js:78]: The position is now stored right away, so even if it changes after the click we already know the original position and the timeout below doesn't influence it.

File: src/UI/controllers/OverlayUIController.js [hunk 4]
```diff
@@ -75,15 +75,19 @@ export class OverlayUIController extends UIController {
...
+
         let delay = 0
-        if (!click && !this.elementIsInViewport(getPosition())) {
+        if (!click && !this.elementIsInViewport(position)) {
             input.scrollIntoView(true)
             delay = 500
         }
+        this.#state = 'parentShown'
```
Comment (GioSensation) [inline src/UI/controllers/OverlayUIController.js:85]: By setting the `parentShown` state right away we avoid multiple fires while the execution is suspended by the timeout.

**Review by alistairjcbrown — APPROVED** [review]
LGTM :shipit:

**Pull request merged**